Database.php

```diff
@@ -3,7 +3,6 @@
 $wgLBFactoryConf = array(
 	'class' => 'LBFactoryMulti',
 	'sectionsByDB' => array(
-	        // 'wiki' => 'c1'
```

Comment:
Re add this please.

Reply:
This isn’t needed

Reply:
Yes it is for reference as we need docs.